src/libraries/System.Private.Uri/src/System/Uri.cs

```diff
@@ -4977,19 +4966,19 @@ private static string CombineUri(Uri basePart, string relativePart, UriFormat ur
             char c1 = relativePart[0];
 
             //check a special case for the base as DOS path and a rooted relative string
-            if (basePart.IsDosPath &&
-                (c1 == '/' || c1 == '\\') &&
-                (relativePart.Length == 1 || (relativePart[1] != '/' && relativePart[1] != '\\')))
+            if (basePart.IsDosPath
+                && (c1 == '/' || c1 == '\\')
+                && (relativePart.Length == 1 || (relativePart[1] != '/' && relativePart[1] != '\\')))
```

Comment:
Why are you changing all of these from end of the previous line to beginning of the next line?

Reply:
No real reason, reverting